include/libmw/node.h

```diff
@@ -100,6 +100,16 @@ MWIMPORT libmw::StateRef SnapshotState(const libmw::CoinsViewRef& view);
 /// <throws>ValidationException if consensus rules are not met.</throws>
 MWIMPORT void CheckTransaction(const libmw::TxRef& transaction);
 
+/// <summary>
+/// Validation of the MWEB transaction inputs against the given CoinsView.
+/// This validates that the inputs are unspent, and their maturity is reached.
+/// </summary>
+/// <param name="transaction">The MWEB transaction to validate. Must not be null.</param>
+/// <param name="view">The CoinsView to validate against. Must not be null.</param>
+/// <param name="nSpendHeight">The height at which the transaction is included.</param>
+/// <throws>ValidationException if consensus rules are not met.</throws>
+MWIMPORT void CheckTxInputs(const libmw::TxRef& transaction, const libmw::CoinsViewRef& view, int nSpendHeight);
```

Comment:
nit: The other APIs all expect the view first. Let's switch the order of transaction and view to be consistent.